TeXLive commands make4ht, tex4ebook fail to process images

### Describe the bug
I'm trying to produce html and epub files from a LaTeX document with the *ht* tools and it looks like they don't work properly.

Given the following simple input, with a PDF file pic.pdf, 
```latex
\documentclass[11pt,a4paper,twoside]{article}
\usepackage{graphicx}
\begin{document}

\section{Introduction}

Look at this Figure~\ref{fig:pic}

\begin{figure}
  \centering
  \includegraphics{pic}
  \caption{Pic.}
  \label{fig:pic}
\end{figure}
\end{document}
```
the commands `make4ht`and `tex4ebook` give the following errors 

```bash
$ make4ht test.tex "mathjax"
[STATUS]  make4ht: Conversion started
[STATUS]  make4ht: Input file: test.tex
sh: line 1: rungs: command not found
[STATUS]  make4ht: Conversion finished
```

```bash
$ tex4ebook test.tex mathml
[STATUS]  tex4ebook: Conversion started
[STATUS]  tex4ebook: Input file: test.tex
sh: line 1: rungs: command not found
[ERROR]   mkutils: File pic-.png doesn't exist
cp: cannot stat 'pic-.png': No such file or directory
sh: line 1: tidy: command not found
[WARNING] exec_epub: tidy command seems missing, you should install it in order
  to make valid epub file
[STATUS]  tex4ebook: Conversion finished
```
In both cases the output is produced, but the image is missing.

### Notify maintainers

@veprbl @vcunat 
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.60, NixOS, 23.05 (Stoat), 23.05.4662.34bdaaf1f0b7`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.6`
 - channels(beffa): `""`
 - channels(root): `"nixos-23.05"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

--- a/pkgs/tools/typesetting/tex/texlive/combine.nix
+++ b/pkgs/tools/typesetting/tex/texlive/combine.nix
@@ -297,6 +297,7 @@ in (buildEnv {
     # install (wrappers for) scripts, based on a list from upstream texlive
   ''
     source '${bin.core.out}/share/texmf-dist/scripts/texlive/scripts.lst'
+    texmf_scripts+=' texlive/rungs.lua' # TL2022: file was renamed from rungs.tlu to rungs.lua but not updated in scripts.lst
     for s in $texmf_scripts; do
       [[ -x "$TEXMFDIST/scripts/$s" ]] || continue
       tName="$(basename $s | sed 's/\.[a-z]\+$//')" # remove extension

